src/frontends/tensorflow/src/helper_ops/unique.hpp

```diff
@@ -36,6 +36,10 @@ class Unique : public ov::frontend::tensorflow::InternalOperation {
         set_output_type(1, out_idx, ov::PartialShape({ov::Dimension::dynamic()}));
     }
 
+    ov::element::Type get_out_idx() const {
+        return out_idx;
+    }
+
 private:
     ov::element::Type out_idx;
```

Comment:
The name is misleading... This is not idx it is type of idx? Please change name of the variable and of the corresponding function. Also it is a good idea to prepend variable name with `m_`.

Reply:
It is aligned with TensorFlow specification: https://www.tensorflow.org/api_docs/python/tf/unique

Reply:
Anyway, I renamed:) I also dislike TF version.